v1.1.1.md

```diff
@@ -0,0 +1,93 @@
+# Harvester v1.1.1 Release Note
+
+The documentation is available at https://docs.harvesterhci.io/v1.1
+
+# Downloads
+
+## ISO
+:cd: [harvester-v1.1.1-amd64.iso](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-amd64.iso)
+
+## Additional Artifacts
+:file_folder: [harvester-v1.1.1-initrd-amd64](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-initrd-amd64)
+:file_folder: [harvester-v1.1.1-vmlinuz-amd64](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-vmlinuz-amd64)
+:file_folder: [harvester-v1.1.1-rootfs-amd64.squashfs](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-rootfs-amd64.squashfs)
+:white_check_mark: [harvester-v1.1.1-amd64.sha512](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-amd64.sha512)
+:memo:  [harvester-v1.1.1-version.yaml](https://releases.rancher.com/harvester/v1.1.1/version.yaml)
+
+# Install
+Harvester can be installed with the [ISO](https://docs.harvesterhci.io/v1.1/install/iso-install/) image or via [PXE](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/) boot.
+
+# Highlights
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
+* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
```

Comment:
do we require to shut down all VMs upgraded from v1.1.0?

Reply:
No need. fixed!